### PR TITLE
fix: typos across project

### LINKED ISSUE #3735: Typos present in project.
There are various typos across the project.

### PR DESCRIPTION
**Pull request by snitin315**
Resolve #3735 

# Description

Fixed Typos Across Project.

### CODE REVIEW

**Comment by znarf** [comment]
Thank you! Have you used an automated tool for that?

Can you run `npm run langs:update` and commit the result?

**Comment by snitin315** [comment]
> Thank you! Have you used an automated tool for that?

Yes, I used [`misspell`](https://github.com/client9/misspell) to find & correct typos.
 
> Can you run `npm run langs:update` and commit the result?

Done.

File: components/expenses/__tests__/Expenses.test.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ describe('Expenses component', () => {
       expect(component.find('[data-cy="mark-expense-as-paid-btn"]')).not.toBeFalsy();
 
       // make sure none are disabled
-      expect(component.find('[data-cy="expense-actions"] button[disabled]').lenght).toEqual(undefined);
+      expect(component.find('[data-cy="expense-actions"] button[disabled]').length).toEqual(undefined);
```
Comment (snitin315) [inline components/expenses/__tests__/Expenses.test.js]: > Also, one test is impacted by the change, could you look into it?

this test is failing, but I think it should be `length`. So I guess code should be reviewed.

Comment (znarf) [reply components/expenses/__tests__/Expenses.test.js]: Yes, I agree, better remove the change and create a separated Issue or PR.

**Comment by snitin315** [comment]
@znarf Done :100: 

I have one query as well that how can I be a member of this organization on GitHub?

**Comment by znarf** [comment]
Thank you @snitin315 !

Would that make sense to integrate that as a CI job?

**Comment by snitin315** [comment]
Yes, should be integrated with CI to avoid future typos.

**Pull request merged**